packages/skeleton/src/skeleton.tsx

```diff
@@ -137,10 +137,11 @@ export const SkeletonText: React.FC<SkeletonTextProps> = (props) => {
     ...rest
   } = props
 
-  const numbers = range(noOfLines)
+  const noOfLinesValue = useBreakpointValue(typeof noOfLines === 'number' ? [noOfLines] : noOfLines) || defaultNoOfLines;
```

Comment:
the `|| defaultNoOfLines` case can't ever happen, can it? if explicitly passed as `undefined`, it would be `defaultNoOfLines` so you're always in a situation of `is number` or `is array`?